HW-11/Homework-11.py

```diff
@@ -0,0 +1,60 @@
+# decorator for any function:
+def my_new_decorator(func):
+    def the_wrapper_around_the_original_function(*args, **kwargs):
+        from datetime import datetime
```

Comment:
It is better to put this line at the top of the file. We will talk about it at the next lesson